sdk/textanalytics/Azure.AI.TextAnalytics/tests/RecognizeHealthcareEntitiesTests.cs

```diff
@@ -0,0 +1,230 @@
+            "Can cause rapid or irregular heartbeat, delirium, panic, psychosis, and heart failure."
+        };
+
+        private static List<TextDocumentInput> batchDocuments = new List<TextDocumentInput>
+        {
+            new TextDocumentInput("1", "Subject is taking 100mg of ibuprofen twice daily")
+            {
+                 Language = "en",
+            },
+            new TextDocumentInput("2", "Can cause rapid or irregular heartbeat, delirium, panic, psychosis, and heart failure.")
+            {
+                 Language = "en",
+            }
+        };
+
+        [Test]
+        public async Task RecognizeHealthcareEntitiesTest()
+        {
+            TextAnalyticsClient client = GetClient();
+            string document = singleEnglish;
+
+            HealthcareOperation operation = await client.StartHealthcareAsync(document);
+
+            await operation.WaitForCompletionAsync();
+
+            RecognizeHealthcareEntitiesResultCollection resultCollection = operation.Value;
+
+            Assert.AreEqual(1, resultCollection.Count);
+
+            var entitiesList = new List<string> { "100mg", "ibuprofen", "twice daily" };
+            foreach (DocumentHealthcareResult result in resultCollection)
+            {
+                foreach (HealthcareEntity entity in result.Entities)
+                {
+                    Assert.IsTrue(entitiesList.Contains(entity.Text));
+                }
+            }
+        }
+
+        [Test]
+        public async Task RecognizeHealthcareEntitiesWithLanguageTest()
+        {
+            TextAnalyticsClient client = GetClient();
+            string document = singleEnglish;
+
+            HealthcareOperation operation = await client.StartHealthcareBatchAsync(new List<string>() { document }, "en");
+
+            await operation.WaitForCompletionAsync();
+
+            RecognizeHealthcareEntitiesResultCollection resultCollection = operation.Value;
+
+            foreach (DocumentHealthcareResult result in resultCollection)
+            {
+                Assert.AreEqual(3, result.Entities.Count);
+                Assert.IsNotNull(result.Id);
+            }
+        }
+
+        [Test]
+        public async Task RecognizeHealthcareEntitiesWithTopParameter()
+        {
+            TextAnalyticsClient client = GetClient();
+
+            HealthcareOptions options = new HealthcareOptions()
+            {
+                Top = 1,
+                Skip = 0,
+                IncludeStatistics = true
+            };
+
+            HealthcareOperation operation = await client.StartHealthcareBatchAsync(batchDocuments, options);
+
+            await operation.WaitForCompletionAsync();
+
+            RecognizeHealthcareEntitiesResultCollection resultCollection = operation.Value;
+
+            Assert.AreEqual(1, resultCollection.Count);
+            Assert.AreEqual(3, resultCollection[0].Entities.Count);
+            Assert.IsNotNull(resultCollection[0].Id);
```

Comment:
would it be good to also verify that the `Id` is 1?

Reply:
its is "0"